qcelemental/checkup_data/cfour_primary_masses.py

```diff
@@ -1,118 +1,117 @@
 # DATA ATMSS from ~line 150 of cfour/joda/pertable.f
 
-# fmt: off
```

Comment:
Fair. I could add it back. The thinking was to move towards a fully `flake8` compliant repo and it's easier if everything is formatted rather than having to annotate for formatters and linters. Just less "special cases" throughout the code. Can add back `# fmt:off` stuff if you'd like. Let me know.

Reply:
Reverted files.